content/projects/data-science-specific/statistical-thinking/_index.md

```diff
@@ -42,38 +42,51 @@ Complete the DataCamp courses [Statistical Thinking in Python Part 1](https://ww
+
+```
+├──data
+│   └──MDG_Export_20191227.csv
+├──notebook
+│  └──statistical_thinking.ipynb
+├──README.md
+├──requirements.txt
+└──.gitignore 
+```
 
 ### Questions
 
-1. How many different countries are represented? How many missing values are there by country, year and series?
-2. Who are the top and bottom 5 countries in terms of CO2 emissions in 1990 and what are their emissions? How has this changed by 2011?
-
-  - Do you spot any problems with the data? If so, fix it.
-
-3. Calculate the mean and median CO2 emissions for 1990. Why do you think these values differ?
-4. Calculate the minimum, maximum and interquartile range of the CO2 emissions for 1990. Using this information, and the mean and median, what does this tell you about the distribution of CO2 emissions?
-5. Create a histogram of the CO2 emissions for 1990. Is this what you expected from your answers in questions 3 and 4?
-6. Calculate the standard deviation and standard error of the mean for CO2 emissions in 1990. How is the standard error different from the standard deviation?
-7. Create a line graph to show CO2 emissions in Brazil, Russia, China, India, the USA and South Africa over time. What does the graph tell you about the difference and change in C02 emissions in these countries?
-8. What is the mean and standard deviation for land area covered by forest in 1990? Why do you think the standard deviation is so large?
-9. Create histograms for land area covered by forest and percentage of area protected in 1990. Describe the distributions.
-10. Create a scatterplot with a regression line using `seaborn.regplot` to show the relationship between the proportion of land area covered by forest and the percentage of area protected in 2000.
+1. Load the [MDG data](MDG_Export_20191227.csv) into a pandas DataFrame. You will need the packages `matplotlib`, `numpy`, `seaborn`, `pandas` and `scipy`.
+2. How many different countries are represented? Save this number in a variable `number_of_countries`. 
+3. How many missing values do we have by country, year and series? You need to create 3 DataFrames:
+
+  - `missing_values_by_country_df` with column names `Country` and `missing_values_count`
+  - `missing_values_by_year_df` with column names `Year` and `missing_values_count` 
+  - `missing_values_by_series_df` with column names `Series` and `missing_values_count`
+
+4. Who are the top and bottom 5 countries in terms of CO2 emissions in 1990 and what are their emissions? Create DataFrames called `top_countries_co2_emmissions_df_1990` and  `bottom_countries_co2_emmissions_df_1990` with columns `Country` and `co2_emissions` and order the data from highest to lowest for `top_countries_co2_emmissions_df_1990` and from lowest to highest for `bottom_countries_co2_emmissions_df_1990`. How have these emissions changed by 2011?
```

Comment:
Instead of just this: "How have these emissions changed by 2011?"
Let's say:

Create similarly named dataframes for the emissions in 2011. How have these emissions changed compared with 1990?